server/server_config.go

```diff
@@ -0,0 +1,19 @@
+package server
+
+import (
+	"time"
+)
+
+type ServerConfig struct {
+	ReadHeaderTimeout time.Duration
+	ReadTimeout       time.Duration
+}
```

Comment:
Why not just introduce separate server Options? like:

```Golang
wsServer := server.NewServer(
    ":80",
    server.WithReadTimeout(60 * time.Second),
    server.WithReadHeaderTimeout(5 * time.Second),
)
```

Reply:
Can but this list with probably increase over time